java/src/test/java/org/rocksdb/WriteBatchThreadedTest.java

```diff
@@ -60,12 +60,15 @@ public void threadedWrites() throws InterruptedException, ExecutionException {
         @Override
         public Void call() throws RocksDBException {
           final WriteBatch wb = new WriteBatch();
-          for (int i = offset; i < offset + 100; i++) {
-            wb.put(ByteBuffer.allocate(4).putInt(i).array(),
-                "parallel rocks test".getBytes());
+          try {
```

Comment:
Perhaps it would be better to use ARM, e.g.:

```java
try(final WriteBatch wb = new WriteBatch();
        final WriteOptions writeOptions = new WriteOptions()) {
    for (int i = offset; i < offset + 100; i++) {
        wb.put(ByteBuffer.allocate(4).putInt(i).array(),
                "parallel rocks test".getBytes());
    }
    db.write(writeOptions, wb);
}
```